ajax/updateTranslationValue.php

```diff
@@ -43,6 +43,7 @@
 $matching_field = null;
 
 if (isset($_POST['itemtype'], $_POST['field']) && is_a($_POST['itemtype'], CommonDropdown::class, true)) {
+    /** @var CommonDropdown $itemtype */
```

Comment:
Probably not needed if the PHPDoc changes from `main` for `getItemForItemtype` are applied.